airbyte-cdk/python/airbyte_cdk/logger.py

```diff
@@ -22,12 +22,59 @@
 # SOFTWARE.
 #
 
-
+import logging
+import sys
 import traceback
 
 from airbyte_cdk.models import AirbyteLogMessage, AirbyteMessage
 
 
+TRACE_LEVEL_NUM = 5
+
+
+def get_logger():
+    logging.setLoggerClass(AirbyteNativeLogger)
+    logging.addLevelName(TRACE_LEVEL_NUM, "TRACE")
+    logger = logging.getLogger('airbyte.source.zabbix')
+    logger.setLevel(TRACE_LEVEL_NUM)
+    handler = logging.StreamHandler(stream=sys.stdout)
+    handler.setFormatter(AirbyteLogFormatter())
+    logger.addHandler(handler)
+    return logger
+
+
+class AirbyteLogFormatter(logging.Formatter):
+    def format(self, record):
+        message = super().format(record)
+        log_message = AirbyteMessage(type="LOG", log=AirbyteLogMessage(level=record.levelname, message=message))
+        return log_message.json(exclude_unset=True)
+
+
+class AirbyteNativeLogger(logging.Logger):
+    def __init__(self, name):
+        logging.Logger.__init__(self, name)
+        self.valid_log_types = {"FATAL": 50, "ERROR": 40, "WARN": 30, "INFO": 20, "DEBUG": 10, "TRACE": 5}
```

Comment:
why numbers here?
please use `addLevelName` to add `TRACE` level
and use `getLevelName` to get enum value from the string